test/parallel/test-bootstrap-modules.js

```diff
@@ -11,4 +11,4 @@ const list = process.moduleLoadList.slice();
 
 const assert = require('assert');
 
-assert(list.length <= 74, list);
+assert(list.length <= 75, list);
```

Comment:
This should not be required anymore, right?

Reply:
*shouldn't be*, but on my ubuntu image, the test fails on master *also* without this change:

```
$ ./node test/parallel/test-bootstrap-modules.js
assert.js:338
    throw err;
    ^

AssertionError [ERR_ASSERTION]: Internal Binding contextify,Internal Binding worker,NativeModule events,Internal Binding trace_events,NativeModule internal/safe_globals,NativeModule internal/async_hooks,NativeModule internal/errors,Internal Binding uv,Binding buffer,Internal Binding async_wrap,Binding config,Binding icu,NativeModule util,NativeModule internal/validators,NativeModule internal/encoding,NativeModule internal/util,Binding constants,Internal Binding util,NativeModule internal/util/types,Internal Binding types,NativeModule buffer,NativeModule internal/buffer,NativeModule internal/process/per_thread,NativeModule internal/process/main_thread_only,NativeModule internal/process/stdio,NativeModule assert,NativeModule internal/assert,NativeModule fs,NativeModule path,NativeModule internal/constants,Binding fs,NativeModule internal/fs/utils,NativeModule internal/url,NativeModule internal/querystring,Binding url,NativeModule internal/process/warning,NativeModule internal/process/next_tick,NativeModule internal/process/promises,NativeModule internal/fixed_queue,Internal Binding performance,NativeModule internal/inspector_async_hook,Binding inspector,Internal Binding options,NativeModule timers,Internal Binding timers,NativeModule internal/linkedlist,NativeModule internal/priority_queue,NativeModule internal/timers,NativeModule internal/modules/cjs/loader,NativeModule vm,NativeModule internal/modules/cjs/helpers,NativeModule console,Internal Binding tty_wrap,NativeModule tty,NativeModule net,NativeModule stream,NativeModule internal/streams/pipeline,NativeModule internal/streams/end-of-stream,NativeModule internal/streams/legacy,NativeModule _stream_readable,NativeModule internal/streams/buffer_list,NativeModule internal/streams/destroy,NativeModule internal/streams/state,NativeModule _stream_writable,NativeModule _stream_duplex,NativeModule _stream_transform,NativeModule _stream_passthrough,NativeModule internal/net,Internal Binding cares_wrap,Internal Binding stream_wrap,Internal Binding tcp_wrap,Internal Binding pipe_wrap,NativeModule internal/stream_base_commons,NativeModule internal/tty,Internal Binding signal_wrap
    at Object.<anonymous> (/home/james/node/node/test/parallel/test-bootstrap-modules.js:14:1)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
    at startup (internal/bootstrap/node.js:270:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:746:3)
```

Reply:
Ah, the failure only occurs when running the test directly as in the above example, running it through the test suite causes it to pass because there is a variance in the number of bootstrapped modules based on how the node.js process is actually launched.

Reply:
That makes sense. I doubt that people actually normally trigger it individually, so I would rather keep it at 74.

Reply:
On the contrary, I trigger tests manually quite frequently as part of my regular workflow. I will be looking at a possible way to refactor the test so that it works regardless of how it is run.